packages/aws-cdk/lib/api/hotswap/s3-bucket-deployments.ts

```diff
@@ -116,6 +118,7 @@ function stringifyObject(obj: any): any {
 
 class EmptyHotswapOperation implements HotswapOperation {
   readonly service = 'empty';
+  readonly resourceNames = ['empty'];
```

Comment:
Let's make this just empty instead:

```suggestion
  public readonly resourceNames = [];
```